packages/documentation-site/patternfly-docs/content/design-guidelines/content/error-messages.md

```diff
@@ -2,62 +2,78 @@
 id: Error messages
 section: UX writing
 ---
+
+## Error message structure
+
 A user typically sees an error message when they attempt to perform an action but cannot continue because something isn’t right. 
 
-Make your error messages brief yet descriptive. A useful pattern to follow is Description, Reason, and Resolution:
-- **Description**: What happened? (Example: The user's login failed.)
-- **Reason**: Why did it happen? (Example: SSH key isn't allowing auto-login.)
-- **Resolution**: How can it be resolved? (Example: Manually log in to the host.)
+Make your error messages brief yet descriptive. A useful pattern to follow is to include a description, reason, and resolution:
+- **Description**: What happened? 
+    - For example, if the user's login failed.
+- **Reason**: Why did it happen? 
+    - For example, an SSH key doesn't allow auto-login.
+- **Resolution**: How can it be resolved?
+    - For example, a user may need  to manually log in to the host.
+    
+Combine your description, reason, and resolution with a title to create a strong error message. For example:
 
-Combined, these parts create this error message:  
+_**Login failed**_ (description)
 
-_**Login failed**_ 
+_The SSH key for auto-login is either not available, is unauthorized, or is password protected._ (reason)
 
-_The SSH key for auto-login is either not available, is unauthorized, or is password protected._
+_To manually log in to the host, click **Log in.**_ (resolution)
 
-_To manually log in to the host, click **Log in.**_
+## Best practices for writing error messages
 
 Keep these best practices in mind when crafting error messages:
 
-**Don’t blame users**. A user should never feel like the error is their fault. Avoid language like “You did something wrong.” Depending on your message, you may need to use the passive voice instead of the active voice so that you don't assign blame to the user.   
+#### **Don’t blame users**. 
```

Comment:
All these sub sections are bolded on top of being headers, so they're showing up extra bold in the guidelines. I don't necessarily find this to be a bad thing, but it's not really consistent with our headers everywhere else, so I would maybe unbold

Reply:
Also, they're all headers but end with periods. I would remove the period, no?

Reply:
yes true - will update those! re punctuation in headings, do you think maybe that's a "titles and headings" section that should be added to the punctuation page (or somewhere else)? I don't know if we mention excluding certain punctuation from headings anywhere currently, but it could be worth writing down somewhere?

Reply:
Yeah, might as well if you think clarifying that is useful!